Added K8s Example

rhjensen79 opened the pull request:
Great job you have done. Just what i was looking for.
I was in the process of creating this myself, but since yours was a really great working example, i added a k8s example to it as well + some minor fixes.

lpasselin commented:
wow thanks a lot for the PR! I really wasn't expecting this!
Thanks for the minor corrections and the k8 example.

If you have some time, can you add a script to launch the k8 example? If you prefer, we can merge right away.
I will do the same in a future PR for the docker-compose examples.

lpasselin commented on README.md:
```diff
@@ -13,6 +13,12 @@ As explained in the blog post, uses a docker-compose service to add the containe
+## K8S
+
+Simple K8S example.
+Build and push the image to a registry, and change to image name, in the deployment.yaml file.
```
```suggestion
Build and tag the image (or push to a registry). Change `YOUR_IMAGE` in the deployment.yaml file.
```

rhjensen79 replied:
Agree on that change

rhjensen79 commented:
Np at all.
I wanted to create it myself, but your repo and code, looked so nice, that it was a pleasure to add to that.

Happy to do some automation around it. 
I will also some actions, with auto build of images, so they are easy to consume. 

Expect to see a couple more PR :-)

rhjensen79 commented:
Sorry closed it by mistake. 

Let me know if you need more in the PR. I will create another for the other changes.

rhjensen79 commented:
Added a bit of automation, and updated the documentation.
Deployment don't work, until you approve the github action, and run it, so a new container image is created in your repo.

lpasselin commented:
I don't have a lot of experience with github actions but would like to experiment first to make sure everything works and is clean. Let's merge in a new dev branch to play with the github actions. I'll add you as maintainer of the repo to simplify things.

rhjensen79 commented:
Deal. I will update the action to work on the dev branch as well, and only trigger when changes are made in the k8s folder.
